lib/fake_sqs/test_integration.rb

```diff
@@ -10,11 +10,11 @@ def initialize(options = {})
     end
 
     def host
-      option :sqs_endpoint
+      URI(option :endpoint).host
```

Comment:
Since endpoint is now required to be a fully qualified URI, we can use the URI class to parse out the host and port.